clayui.com/src/components/LayoutNav/LayoutNav.js

```diff
@@ -49,27 +51,33 @@ export default () => {
 					<ul className="ml-auto navbar-nav">
 						<li className="nav-item">
 							<Link
-								activeStyle={{color: '#b3472d'}}
 								className="nav-link"
-								partiallyActive
-								to="/docs/"
+								style={{
+									color: pathname.match(/docs\//g)
+										? '#b3472d'
+										: '',
+								}}
+								to="/docs/get-started/index.html"
 							>
 								<span className="c-inner" tabIndex="-1">
 									{'Docs'}
 								</span>
 							</Link>
 						</li>
 						<li className="nav-item">
-							<Link
-								activeStyle={{color: '#b3472d'}}
+							<a
 								className="nav-link"
-								partiallyActive
-								to="/blog"
+								href="/blog"
+								style={{
+									color: pathname.match(/blog\//g)
+										? '#b3472d'
+										: '',
+								}}
 							>
 								<span className="c-inner" tabIndex="-1">
 									{'Blog'}
 								</span>
-							</Link>
+							</a>
```

Comment:
Hey @pat270 I think that replacing this with a href will have a redirection made by the browser instead of redirecting via JS, I think it is safe to keep it even with `console.log` because we make this rule here https://github.com/liferay/clay/blob/master/clayui.com/gatsby/createPages.js#L210-L215.

Reply:
@matuzalemsteles I can change it back. This doesn't fix the issue. There are still warnings on the development site. It might be a bug in the gatsby version we are using. I tried upgrading to a newer version but kept running into build issues.

Reply:
@pat270 no problem, I also think it could be a bug, because we are creating a redirect for this route, in theory I should not give this warning. I think we can open an issue to update Gatsby to a newer version and resolve these compilation errors in another PR.